src/funk/fd_funk_rec.h

```diff
@@ -297,6 +297,50 @@ fd_funk_rec_remove( fd_funk_t *               funk,
                     fd_funk_rec_t **          rec_out,
                     ulong                     erase_data );
 
+/* fd_funk_rec_modify_try queries the in-prep transaction pointed to
+   by txn for the record whose key matches the key pointed to by the
+   key. If the query is successful, then we will acquire a lock on the
+   corresponding hash chain. Any updates made to the record can be
+   published into the funk transaction with a call to
+   fd_funk_rec_modify_publish. */
```

Comment:
```suggestion
  fd_funk_rec_modify_fini. */
```
`fd_funk_rec_modify_test` is a bit unclear as to what the function does. Usually `_test` implies checking for lock contention or something.
what about `fd_funk_rec_modify_fini`?

Reply:
it adheres to the naming convention within `fd_map_chain_para`. i'm open to either though